pandas/_libs/groupby.pyx

```diff
@@ -388,10 +388,10 @@ def group_fillna_indexer(ndarray[int64_t] out, ndarray[intp_t] labels,
 
 @cython.boundscheck(False)
 @cython.wraparound(False)
-def group_any_all(int8_t[::1] out,
-                  const int8_t[::1] values,
-                  const intp_t[:] labels,
-                  const uint8_t[::1] mask,
+def group_any_all(int8_t[:, ::1] out,
+                  const int8_t[:, :] values,
+                  const intp_t[::1] labels,
```

Comment:
If we have a contiguity guarantee for `labels` here, would that also be the case for other groupby algos? Just wondering because I think the others don't specify `labels` contiguity, so might be able to gain small speedup by doing that.

Reply:
i think thats the case, yes.  merits double-checking, especially since some builds are failing